// The Firmament project
// Copyright (c) 2013 Malte Schwarzkopf <malte.schwarzkopf@cl.cam.ac.uk>
//
// A runtime sample of a task's resource demands.

syntax = "proto3";

package firmament;

message TaskPerfStatisticsSample {
  uint64 task_id = 1;
  uint64 timestamp = 2;
  uint64 vsize = 3;
  uint64 rsize = 4;
  uint64 sched_run = 5;
  uint64 sched_wait = 6;
  bool completed = 7;
  string hostname = 8;
}
